omniduct/wrappers/redis.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import absolute_import
+
+import redis
+from omniduct.utils.debug import logger
+from .base import WrapperClient
+
+from omniduct.utils.magics import (MagicsProvider, process_line_arguments,
+                                   process_line_cell_arguments)
+
+
+class RedisClient(WrapperClient, MagicsProvider):
+    """
+    This Duct connects to a redis database server using the `redis` python library.
+    """
+    PROTOCOLS = ['redis']
+    DEFAULT_PORT = 6379
+
+    def _init(self):
+        self._redis_connection = None
+
+    def _connect(self):
+        self._redis_connection = redis.Redis(self.host, self.port)
+
+    def _is_connected(self):
+        return hasattr(self, '_redis_connection') and self._redis_connection is not None
+
+    def _disconnect(self):
+        logger.info('Disconnecting from Redis database ...')
+        self._redis_connection = None
+
+    @property
```

Comment:
I'm thinking this should really be a class property but not sure how to make an abstract class property?